onnxruntime/hosting/http/session.cc

```diff
@@ -93,31 +93,34 @@ void HttpSession::Send(Msg&& msg) {
 }
 
 template <typename Body, typename Allocator>
-void HttpSession::HandleRequest(boost::beast::http::request<Body, boost::beast::http::basic_fields<Allocator> >&& req) {
+void HttpSession::HandleRequest(http::request<Body, http::basic_fields<Allocator> >&& req) {
   HttpContext context{};
-  context.request = req;
+  context.request = std::move(req);
 
-  std::string path = req.target().to_string();
-  std::string model_name;
-  std::string model_version;
-  std::string action;
+  std::string path = context.request.target().to_string();
+  std::string model_name, model_version, action;
```

Comment:
If clang-format does not complain, I'm ok with multiple variables define in the same line. :-)

Reply:
Haha it does not.